### ISSUE TITLE
Updated Views Bulk Operation for 1.0.0 release to a supported release?

### ISSUE THREAD

**Issue by theMusician** [esmero/archipelago-deployment-live#54]
In our RC3 instance, Views Bulk Operation (VBO) reports as unsupported in the Drupal Status report. 

https://github.com/esmero/archipelago-deployment-live/blob/ac2a2155256466cf8fdd06c227e0415bd9471487/drupal/composer.default.json#L78

$ composer require 'drupal/views_bulk_operations:^4.1'

[4.1.2 is the latest release available](https://www.drupal.org/project/views_bulk_operations/releases/4.1.2). This update I imagine works but am hesitant to test it quite yet until I have documented all of the places VBO is used throughout Archipelago.

**Comment by DiegoPino** [comment]
Yes, but to play it right we need also this https://www.drupal.org/project/moderated_content_bulk_publish/issues/3280584 patch that needs to go at the same time. Also AMI 0.4.0 already depends on 4.1 so we need to make the complete revamp. Will also mark this one as part of the release 1.0.0 composer.json (will have it by tomorrow) changes. Thanks!
